product_simple_seasonality/views/product_template.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2024 Akretion
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<odoo>
+
+    <record model="ir.ui.view" id="product_template_form_view">
+        <field
+            name="name"
+        >product.template.form (in product_campaign_seasonality)</field>
+        <field name="model">product.template</field>
+        <field name="inherit_id" ref="product.product_template_form_view" />
+        <field name="arch" type="xml">
+            <xpath expr="//group[@name='description']" position="inside">
+                <field name="seasonality_ids" />
```

Comment:
Maybe we can have a widget with checkbox like route_ids field in product inventory tab ?

Reply:
Maybe we can move towards optionnal_product_ids

Reply:
I'd like more the many2manytags widget :)

Reply:
after Upsell